pom.xml

```diff
@@ -205,6 +206,36 @@
                 <module>tcks</module>
             </modules>
         </profile>
+        <profile>
+            <id>gib</id>
```

Comment:
I am not totally certain on this, but can we name it `incremental` (and the activation property)? I think it's more intuitive (although `gib` is shorter to type, but I think readability is important)

The reason why I am, uncertain is that I see that you have other parameters when you use `-Dgib` that also starts with `-Dgib.`, so maybe it makes sense.

I'm curious to know what others think.

Reply:
Yeah sure, why not.

Reply:
@aloubyansky @gsmet @maxandersen what would you prefer? `-Dgib` or `-Dincremental`?
All other `gib.*` properties must stay this way since those are GIB built-ins.

Reply:
"-Dmore-quickly" ? :) 

just kidding.

"-Dincremental" works for me. better to have profile be about the intent rather than the technology used.

Reply:
Ok, renamed to `incremental`.